nextflow.config

```diff
@@ -134,6 +131,7 @@ profiles {
     debug { process.beforeScript = 'echo $HOSTNAME' }
     conda {
         params.enable_conda    = true
+        conda.enabled          = true
```

Comment:
The current version of the modules installed on the pipeline still uses the old syntax `conda (params.enable_conda ? "bioconda::tool=version" : null)`, won't then this change break the possibility of running the pipeline conda?

Reply:
Nope. It shouldn't do. It will just mean that it works with later versions of Nextflow.

Reply:
I dont know why I have read the change as you also deleted `params.enable_conda = true` :sweat: